sympy/plotting/tests/test_plot.py

```diff
@@ -1337,3 +1337,20 @@ def test_plot3d_parametric_surface_arguments():
     assert p[1].ranges[1][1:] == (-4, 5)
     assert p[1].get_label(False) == "test"
     assert p[1].rendering_kw == {}
+
+def test_issue_25190():
+    if not matplotlib:
+        skip("Matplotlib not hte default backend")
```

Comment:
Is there an error in the message here? Should not be `"Matplotlib not in the default backend"` ?

Reply:
Typo, fixed it.